### PR TITLE
#2069 svarcs add static variable reference on regulation mode only wh…

### LINKED ISSUE #2069: SVarC data interface: add static vraibale reference on regulation mode only when stand by automaton exists
This will allow one to use SVarC mdoels without mode handling starting from an iidm

### PR DESCRIPTION
**Pull request by marcochiaramello**
…en stand by automaton exists

Closes #2069

### CODE REVIEW

File: dynawo/sources/Modeler/DataInterface/IIDM/test/TestStateVariable.cpp [hunk 1]
```diff
@@ -329,7 +334,7 @@ TEST(ModelsModelNetwork, TestNetworkCreation) {
     ASSERT_EQ(staticVarCompensators.size(), 1);
     for (std::size_t j = 0, jEnd = staticVarCompensators.size(); j < jEnd; ++j) {
       shared_ptr<StaticVarCompensatorInterface> staticVarCompensator = staticVarCompensators[j];
-      ASSERT_EQ(staticVarCompensator->getRegulationMode(), StaticVarCompensatorInterface::RUNNING_V);
+      ASSERT_EQ(staticVarCompensator->getRegulationMode(), StaticVarCompensatorInterface::STANDBY);
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/DataInterface/IIDM/test/TestStateVariable.cpp:337]: Would be nice to add a test without the standByAutomaton

File: dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp [hunk 2]
```diff
@@ -1374,20 +1374,13 @@ TEST(DataInterfaceIIDMTest, testStaticVarCompensatorIIDM) {
   svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_P, 4.);
   svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_Q, 1);
   svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_STATE, OPEN);
-  svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_REGULATINGMODE, StaticVarCompensatorInterface::RUNNING_V);
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp:1377]: WOould it be possible to add a test with a standByAutomaton? Maybe in private repo?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp (hunk 2 above)
Comment (marcochiaramello) [reply dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp:1377]: yes, I think it is a good idea, but it must be on private repo as private extension

File: dynawo/sources/Modeler/DataInterface/IIDM/test/TestStateVariable.cpp (hunk 1 above)
Comment (marcochiaramello) [reply dynawo/sources/Modeler/DataInterface/IIDM/test/TestStateVariable.cpp:348]: done!

**Review by rosiereflo — APPROVED** [review]

**Comment by rosiereflo** [comment]
There is a code smell that should be addressed

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**